testo.d/xol.test

```diff
@@ -18,7 +18,8 @@ ${PNAME} -i xol -f ${REFERENCE}/doesnotexist -o gpx -F ${TMPDIR}/xol-sample_si.g
 # check error message is what we expected
 # Note that the reference file nonexistent.err includes the file path,
 # so this will fail when starting from a directory other than gpsbabel.
-compare ${REFERENCE}/nonexistent.err ${TMPDIR}/nonexistent.err
+# the message can vary depending on the c runtime, .i.e. msvc and gcc are slightly different.
+compare_with_alternate ${REFERENCE}/nonexistent.err ${REFERENCE}/nonexistent_alternate.err ${TMPDIR}/nonexistent.err
```

Comment:
It's probably OK to just verify that we exited with something other than 0. Is checking $? -gt 0 enough?

Reply:
I don't see compare_with_alternate() in my tree. Does this depend on some other local change?

Reply:
It went into testo last year: https://github.com/gpsbabel/gpsbabel/commit/fbfcbe3d6ad59e2da5a677b435eb0c15bf77e285#diff-69e153e4d7add22f245e24de590eec21